release.yaml

```diff
@@ -24,4 +24,10 @@ rancher-eks-operator:
 - 102.1.2+up1.2.2-rc2
 rancher-eks-operator-crd:
 - 102.1.1+up1.2.0
-- 102.1.2+up1.2.2-rc2
\ No newline at end of file
+- 102.1.2+up1.2.2-rc2
+rancher-gke-operator:
+- 102.0.0+up1.1.5
+- 102.0.1+up1.1.5-rc2
```

Comment:
should it be 1.1.6-rc1? Seems like we already released 1.1.5

Reply:
https://github.com/rancher/charts/pull/2773 seems to be correct, right? We can close current in favor the new